src/App.tsx

```diff
@@ -1,4 +1,14 @@
+import styled from "@emotion/styled";
 import React from "react";
 import { Router } from "./Router";
+import { base } from "./styles/base";
 
-export const App: React.FunctionComponent = () => <Router />;
+export const App: React.FunctionComponent = styled(({ className }) => {
+  return (
+    <div className={`${className} text-gray-800`}>
```

Comment:
Thinking if it make sense to put all the styling here. I'm thinking if a component `BaseStyle` and you can put both the class and the base() inside... wdyt?

Reply:
either way also can, for now maybe lets try embracing tailwind way of setup then we see how it goes